notifier/selfstate/heartbeat/notifier.go

```diff
@@ -20,7 +20,7 @@ func GetNotifier(logger moira.Logger, database moira.Database) Heartbeater {
 
 func (check notifier) Check(int64) (int64, bool, error) {
 	if state, _ := check.db.GetNotifierState(); state != moira.SelfStateOK {
-		check.log.Errorb().
+		check.log.Error().
 			String("error", check.GetErrorMessage()).
 			Msg("Send message")
```

Comment:
hmm, seems, here should be NOT send, see first issue in new sentry :)

Reply:
Fixed